[2025-03-07T18:52:52.509+0000] {processor.py:186} INFO - Started process (PID=491) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:52:52.510+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:52:52.513+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:52:52.513+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:52:52.523+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:52:52.522+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:52:52.524+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:52:52.542+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.044 seconds
[2025-03-07T18:53:22.827+0000] {processor.py:186} INFO - Started process (PID=500) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:53:22.831+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:53:22.835+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:53:22.835+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:53:22.842+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:53:22.841+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:53:22.842+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:53:22.869+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.049 seconds
[2025-03-07T18:53:53.120+0000] {processor.py:186} INFO - Started process (PID=509) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:53:53.121+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:53:53.124+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:53:53.123+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:53:53.134+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:53:53.133+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:53:53.134+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:53:53.161+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.049 seconds
[2025-03-07T18:54:23.439+0000] {processor.py:186} INFO - Started process (PID=518) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:54:23.440+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:54:23.441+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:54:23.441+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:54:23.447+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:54:23.447+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:54:23.447+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:54:23.462+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.031 seconds
[2025-03-07T18:54:53.716+0000] {processor.py:186} INFO - Started process (PID=527) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:54:53.719+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:54:53.720+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:54:53.720+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:54:53.726+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:54:53.725+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:54:53.726+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:54:53.748+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.035 seconds
[2025-03-07T18:55:24.017+0000] {processor.py:186} INFO - Started process (PID=536) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:55:24.019+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:55:24.020+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:55:24.020+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:55:24.026+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:55:24.025+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:55:24.026+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:55:24.050+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.040 seconds
[2025-03-07T18:55:54.316+0000] {processor.py:186} INFO - Started process (PID=545) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:55:54.318+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:55:54.322+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:55:54.321+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:55:54.334+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:55:54.332+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:55:54.334+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:55:54.359+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.051 seconds
[2025-03-07T18:56:24.622+0000] {processor.py:186} INFO - Started process (PID=554) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:56:24.625+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:56:24.627+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:56:24.627+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:56:24.633+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:56:24.633+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:56:24.634+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:56:24.646+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.031 seconds
[2025-03-07T18:56:54.921+0000] {processor.py:186} INFO - Started process (PID=563) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:56:54.923+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:56:54.925+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:56:54.925+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:56:54.933+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:56:54.932+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:56:54.934+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:56:54.951+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.036 seconds
[2025-03-07T18:57:25.223+0000] {processor.py:186} INFO - Started process (PID=572) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:57:25.227+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:57:25.231+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:57:25.230+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:57:25.239+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:57:25.239+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:57:25.240+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:57:25.267+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.049 seconds
[2025-03-07T18:57:55.501+0000] {processor.py:186} INFO - Started process (PID=581) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:57:55.502+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:57:55.503+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:57:55.503+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:57:55.508+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:57:55.507+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:57:55.508+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:57:55.522+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.025 seconds
[2025-03-07T18:58:25.786+0000] {processor.py:186} INFO - Started process (PID=590) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:58:25.789+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:58:25.792+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:58:25.791+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:58:25.799+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:58:25.798+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:58:25.799+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:58:25.826+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.050 seconds
[2025-03-07T18:58:56.098+0000] {processor.py:186} INFO - Started process (PID=599) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:58:56.099+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:58:56.101+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:58:56.101+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:58:56.109+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:58:56.108+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:58:56.110+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:58:56.140+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.054 seconds
[2025-03-07T18:59:26.378+0000] {processor.py:186} INFO - Started process (PID=608) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:59:26.381+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:59:26.383+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:59:26.383+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:59:26.392+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:59:26.391+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:59:26.392+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:59:26.419+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.046 seconds
[2025-03-07T18:59:56.625+0000] {processor.py:186} INFO - Started process (PID=617) to work on /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:59:56.626+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/data-eng-dag-etl/dag.py for tasks to queue
[2025-03-07T18:59:56.627+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:59:56.627+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:59:56.632+0000] {logging_mixin.py:190} INFO - [2025-03-07T18:59:56.631+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/data-eng-dag-etl/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/data-eng-dag-etl/dag.py", line 10, in <module>
    import constants as C
ModuleNotFoundError: No module named 'constants'
[2025-03-07T18:59:56.632+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/data-eng-dag-etl/dag.py
[2025-03-07T18:59:56.652+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/data-eng-dag-etl/dag.py took 0.030 seconds
